src/main/java/com/depromeet/domain/missionRecord/application/MissionRecordService.java

```diff
@@ -150,6 +153,103 @@ public void deleteInProgressMissionRecord() {
     }
 
+    @Transactional(readOnly = true)
+    public MissionStatisticsResponse findMissionStatistics(Long missionId) {
+        Mission mission = findMissionById(missionId);
+        LocalDateTime startedAt = mission.getStartedAt();
+        LocalDateTime finishedAt = mission.getFinishedAt();
+        LocalDateTime today = LocalDateTime.now();
+
+        List<MissionRecord> missionRecords = missionRecordRepository.findAllByMissionId(missionId);
+
+        // 달성률
+        double totalMissionAttainRate =
+                calculateMissionAttainRate(
+                        missionRecords.size(),
+                        Duration.between(startedAt, finishedAt.isBefore(today) ? finishedAt : today)
+                                        .toDays()
+                                + DAYS_ADJUSTMENT);
+
+        // 시간표 생성
+        List<FocusMissionTimeOfDay> timeTable = generateRecordTimeTable(missionRecords);
+
+        // 최대 연속성 계산
+        long maxContinuousSuccessDay =
+                calculateMaxContinuousSuccessDay(startedAt, finishedAt, missionRecords);
+
+        // 전체 번개 스택
+        long totalSymbolStack = calculateTotalSymbolStack(timeTable);
+
+        // 전체 수행시간
+        long sumDuration = calculateSumDuration(timeTable);
+
+        // 전체 수행 시간 (시간)
+        long totalMissionHour = sumDuration / 60;
+
+        // 전체 수행 시간 (분)
+        long totalMissionMinute = sumDuration % 60;
+
+        return MissionStatisticsResponse.of(
+                totalMissionHour,
+                totalMissionMinute,
+                totalSymbolStack,
+                maxContinuousSuccessDay,
+                missionRecords.size(),
+                totalMissionAttainRate,
+                startedAt,
+                finishedAt,
+                timeTable);
+    }
+
+    private List<FocusMissionTimeOfDay> generateRecordTimeTable(
+            List<MissionRecord> missionRecords) {
+        return missionRecords.stream()
+                .map(
+                        record ->
+                                FocusMissionTimeOfDay.of(
+                                        record.getDuration().toMinutes() / 10,
+                                        record.getDuration().toMinutes(),
+                                        record.getStartedAt(),
+                                        record.getFinishedAt()))
+                .toList();
+    }
+
+    private long calculateMaxContinuousSuccessDay(
+            LocalDateTime startedAt, LocalDateTime finishedAt, List<MissionRecord> missionRecords) {
+        long continuousSuccessDay = 1;
+        long maxContinuousSuccessDay = 0;
+        LocalDate previousDate = null;
+
+        for (MissionRecord missionRecord : missionRecords) {
+            LocalDate currentDate = missionRecord.getStartedAt().toLocalDate();
+
+            // startedAt과 finishedAt 사이에 있는 일자일 때만 고려
+            if (currentDate.isAfter(startedAt.toLocalDate())
+                    && currentDate.isBefore(finishedAt.toLocalDate())) {
```

Comment:
```suggestion
            if (!(currentDate.isAfter(startedAt.toLocalDate())
                    && currentDate.isBefore(finishedAt.toLocalDate()))) {
                        return;
                    }
```

early return 해주면 좋을 것 같습니다.